modules/geochemistry/src/utils/GeochemistryActivityCoefficients.C

```diff
@@ -0,0 +1,180 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "GeochemistryActivityCoefficients.h"
+#include "GeochemistryActivityCalculators.h"
+
+GeochemistryActivityCoefficients::GeochemistryActivityCoefficients(
+    ActivityCoefficientMethodEnum method, const GeochemistryIonicStrength & is_calculator)
+  : _method(method),
+    _is_calculator(is_calculator),
+    _ionic_strength(1.0),
+    _sqrt_ionic_strength(1.0),
+    _stoichiometric_ionic_strength(1.0),
+    _num_basis(0),
+    _num_eqm(0),
+    _dhA(0.0),
+    _dhB(0.0),
+    _dhBdot(0.0),
+    _dha(0.0),
+    _dhb(0.0),
+    _dhc(0.0),
+    _dhd(0.0),
+    _dhatilde(0.0),
+    _dhbtilde(0.0),
+    _dhctilde(0.0),
+    _dhdtilde(0.0)
+{
+}
+
+void
+GeochemistryActivityCoefficients::setInternalParameters(
+    Real /*temperature*/,
```

Comment:
Why is temperature not needed?

Reply:
Yep, waiting on that PR of yours to be merged!